examples/lb-http-separate-frontend-and-backend/variables.tf

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type = string
+}
+
+variable "target_size" {
+  type    = number
+  default = 2
+}
+
+variable "group1_region" {
+  type    = string
+  default = "us-west1"
+}
+
+variable "group2_region" {
+  type    = string
+  default = "us-east1"
+}
+
+variable "network_prefix" {
+  type    = string
+  default = "lb-http-separate-frontend-and-backend"
+}
```

Comment:
Let's hard code these in examples rather than exposing as variable

Reply:
Done.